### DISCUSSION TITLE
`not-prose` class not suggested by Tailwind CSS Intellisense

### DISCUSSION THREAD

**Discussion by meesvandongen** [tailwindlabs/tailwindcss#7624] [Help]
**What version of Tailwind CSS IntelliSense are you using?**

For example: v0.7.4

**What version of Tailwind CSS are you using?**

3.0.1

**What package manager are you using?**

pnpm

**What operating system are you using?**

windows

**Reproduction URL**

https://play.tailwindcss.com/4FXzEkbMQu

**Describe your issue**

The [`not-prose` option](https://github.com/tailwindlabs/tailwindcss-typography/blob/master/CHANGELOG.md) is not suggested with tailwind intellisense. Only applicable to typography plugin .

**Comment by simonswiss** [comment]
Hey!

This happens because `not-prose` doesn't actually generate any CSS by itself - it's used as a _flag_ to make another class behave differently.

Since it doesn't output any CSS, this class doesn't really exist by itself in Tailwind CSS. And that's why the Intellisense plugin doesn't suggest it.

The same happens with the `group` class that is used for things like `group-hover`, or the `peer` variant. You can try typing those classes and you'll notice the Intellisense doesn't suggest them to you.

I know it's sort of confusing. Think of this class as an "opt-in flag" instead of a CSS class, and it will make more sense 👍

Moving this to our GitHub discussions as you're definitely not the only one getting confused by this!

**Reply by supto51** [reply]
Hey bro 
Use
Tailwind CSS IntelliSense
v0.7.7
in vs code
